### PR TITLE
Added fix to limit vultr requests down to 2/second (official limits)

### PR DESCRIPTION
**Pull request by bitshark**
Note that this only seems to work when running terraform with -parallelism=1
closes issue #8

### CODE REVIEW

**Comment by squat** [comment]
Hi @bitshark thanks for reporting this issue and submitting a PR. When you say "this only seems to work when running terraform with -parallelism=1" do you mean it breaks terraform with "-parallelism=0?"

File: vultr/config.go [hunk 1]
```diff
@@ -19,7 +20,7 @@ type Client struct {
 
 // Client configures and returns a fully initialized Vultr Client.
 func (c *Config) Client() (interface{}, error) {
-	client := Client{lib.NewClient(c.APIKey, &lib.Options{})}
+	client := Client{lib.NewClient(c.APIKey, &lib.Options{RateLimitation: 2 * time.Second})}
```
Comment (squat) [inline vultr/config.go]: Why did you choose `2 * time.Second`? If the vultr'S default API rate limit is two requests per second, shouldn't this be `0.5 * time.Second`?

**Comment by squat** [comment]
@bitshark ping

**Comment by teh-username** [comment]
Hey @squat, happy to create a new PR with the correction if @bitshark still won't respond.

**Comment by squat** [comment]
@teh-username, please, go for it!

**Comment by squat** [comment]
@bitshark thanks for this PR. I merged #17 in favor of this.

**Pull request closed without merging**